.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @DataDog/apm-dotnet
```

Comment:
are you planning to merge this as is? or do more cleanups before merge?

Reply:
As discussed in the SIG meeting, I want to merge as is to unblock others from contributing. However, I do plan to clean up subsequently.